src/server/api/data.js

```diff
@@ -149,5 +149,41 @@ export default function dataRouter() {
         })
     );
 
+    logger.info("adding the GET /api/filesystem/file/manifest handler");
+    api.get(
+        "/filesystem/file/manifest",
```

Comment:
This is somewhat secondary to this PR, but it would be worth checking the performance of the manifest endpoint, particularly whether we're returning anything we don't need/that might be a performance drain. `content-type` is often less than great, performance-wise, for example.

longer-term conversation, anyway, but I thought I'd mention it; no changes needed in this PR anyway